src/openpgp.js

```diff
@@ -99,7 +99,7 @@ export function destroyWorker() {
 export function generateKey({ userIds=[], passphrase, numBits=2048, unlocked=false } = {}) {
   const options = formatUserIds({ userIds, passphrase, numBits, unlocked });
 
-  if (!util.getWebCryptoAll() && asyncProxy) { // use web worker if web crypto apis are not supported
+  if (asyncProxy) { // use web worker when available
```

Comment:
not that much, but I looked at the saucelabs results where i see practically no difference.

basically for what we gain in the browser crypto, we loose with the other routines running in the main thread; if you try on a mobile or on IE you always the annoying messages that want to kill the browser and you. one day or an other they may succeed.